openml-lightgbm/lightgbm-builder/make.sh

```diff
@@ -35,8 +36,8 @@ DIR=lightgbmlib_build
 if [ ! -d "$DIR" ]; then
   echo "Entering the folder."
   cd make-lightgbm
-  echo "Building LightGBM $VERSION"
-  bash make.sh "$VERSION"
+  echo "Building LightGBM $LIGHTGBM_VERSION as lightgbmlib LIGHTGBMLIB_VERSION"
```

Comment:
Missing a `$` in the last var

Reply:
Thanks! great catch :) Fixed in the latest commit ammend.